src/js/tracks/text-track-display.js

```diff
@@ -243,7 +243,7 @@ class TextTrackDisplay extends Component {
     const tracks = this.player_.textTracks();
     const allowMultipleShowingTracks = this.options_.allowMultipleShowingTracks;
 
-    this.clearDisplay();
+    // this.clearDisplay();
```

Comment:
probably should do this in a backwards compatible way, so, if someone is using an older vttjs, captions don't just accumulate.

Reply:
Looks like vtt.js already removed old things, so, this was never necessary, really. The addition of the else below with clearDisplay is necessary though for when turning all the captions off.
Thus, this line can be removed and we will be backward and forward compatible.